tests/dummy/app/pods/docs/guides/data-fetching/demo-1/component.js

```diff
@@ -20,24 +20,26 @@ export default Component.extend({
   model: readOnly('visit.last.value'),
   activeRoute: readOnly('visitedRoutes.lastObject'),
 
-  routes: {
-    '/posts': {
-      // BEGIN-SNIPPET demo1-posts-route.js
-      // route
-      model() {
-        return this.get('store').findAll('post');
+  routes: computed(function() {
+    return {
```

Comment:
Ember doesn't allow objects on properties anymore, so I made a computed that returned an object/ /shrugguy

Reply:
interesting, must be to prevent the self-troll for object references. Not terrible

Reply:
Yup and memory leaks